docs/index.bs

```diff
@@ -3145,6 +3146,22 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                   1. Let |response| be |requestResponse|'s response.
                   1. If |client| is not null, |response|'s [=response/type=] is "`opaque`", and [=cross-origin resource policy check=] with |request|'s [=request/origin=], |client|, "", and |response|'s [=filtered response/internal response=] returns <b>blocked</b>, then return null.
                   1. Return |response|.
+          1. Else if |source| is {{RouterSource/"race-network-and-fetch-handler"}}, and |request|'s [=request/method=] is \`<code>GET</code>\` then:
+
+              Note: In order to avoid duplicated fetches to |request|, the user agent may reuse |raceNetworkRequestResponse| as the result of |e|'s [=FetchEvent/potential response=], or the fallback request.
+
+              1. Run the following substeps [=in parallel=], but [=abort when=] |controller|'s [=fetch controller/state=] is "<code>terminated</code>" or "<code>aborted</code>":
+                  1. Set |raceFetchController| to the result of [=Fetch|fetching=] |request|.
+
+                      To [=fetch/processResponse=] for |raceNetworkRequestResponse|, run these substeps:
+                          1. If |response| is not null, or |raceNetworkRequestResponse|'s [=response/type=] is "`error`", then:
+                            1. Terminate these substeps.
+                          1. If |raceNetworkRequestResponse|'s [=response/status=] is `200`, then:
+                            1. Set |response| to |raceNetworkRequestResponse|.
+                            1. Return |response|.
+          1. [=If aborted=], then:
```

Comment:
Maybe wrong indent?  You might need another indent here, or this statement is the same level with dispatching behaviors for source.